geode-core/src/test/java/org/apache/geode/internal/DeployedJarJUnitTest.java

```diff
@@ -51,22 +50,14 @@ public void setup() throws Exception {
 
   @Test
   public void validJarContentDoesNotThrow() throws Exception {
-    new DeployedJar(jarFile, JAR_NAME, expectedJarBytes);
-  }
-
-  @Test
-  public void unexpectedContentThrowsException() throws Exception {
-    givenUnexpectedJarFileContents();
-
-    assertThatThrownBy(() -> new DeployedJar(jarFile, JAR_NAME, expectedJarBytes))
-        .isInstanceOf(IllegalStateException.class);
+    new DeployedJar(jarFile, JAR_NAME);
   }
 
   @Test
   public void invalidContentThrowsException() throws Exception {
     byte[] invalidJarBytes = givenInvalidJarBytes();
```

Comment:
I would recommend not declaring and assigning invalidJarBytes as it is not used. Just make the call to givenInvalidBytes().

Even better, rename givenInvalidBytes() as a 'void creatreInvalidJarBytes()'